test/graphql/resend_transaction_mutation_test.rb

```diff
@@ -0,0 +1,159 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ResendTransactionMutationTest < ActiveSupport::TestCase
+  QUERY = <<~EOS
```

Comment:
It is actually separate, but here to test resendTransaction mutation I have to run watchTransaction mutation at first in order to have original tx to refer to. What would you recommend here?

Reply:
What do you mean by only test the query works?

Reply:
Refactored it with FactoryBot

Reply:
Sorry for not being clear there. Ideally, each test just calls the corresponding query and checks specific outputs.

My primary issue was that the test was calling other endpoints/query, instead of testing if the query/interface just works as expected. I was thinking majority of the business validation should be moved to `WatchingTransaction` similar to `kyc.rb` and that query and mutation shoud just pattern match against them.

Reply:
So is it fine now?